src/components/ProgramForm/ThreeColProgram/List/index.css

```diff
@@ -163,3 +162,59 @@
   text-align: center;
   transform: translate(-50%, -50%);
 }
+
+/* RECOMMENDER */
+.recommender {
+  position: relative;
+  display: flex;
+  gap: 8px;
+
+  margin: 8px;
+  padding: 14px 14px 14px 0;
+  background-color: #dbfddc;
+  border-radius: 12px;
+}
+.recommender__title {
+  color: #2dac34;
+  font-weight: 600;
+  font-size: 12px;
+}
+.recommender__paragraph {
+  font-size: 10px;
+}
+.recommender__paragraph--span {
+  color: #2dac34;
+  font-weight: 600;
+}
+.recommender__button {
+  padding: 5px 12px;
+  color: white;
+  font-weight: 600;
+  font-size: 10px;
+  background-color: #2dac34;
+  border-color: #2dac34;
+  border-radius: 2px;
+}
+.recommender__button:hover {
+  color: white;
```

Comment:
I would avoid `keyword colors` - create custom CSS var in recommender block (for this scope only).